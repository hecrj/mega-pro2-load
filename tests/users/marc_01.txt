8
17
50
127
60
35
40
500
360

8
3 5 6 0 0 8 0 0 1 4 0 7 0 0 2 0 0

10
4 3 6 7 8
20
3 1 3 8
35
3 5 6 7
15
1 1
25
1 3
5
0
5
1 1
35
2 6 7

-1 5 0

-2 1 

-5 1 3 

-4 6 1 3 0 

-1 3 5

-5 0 6

-1 1 10 

-3 11 

-1 7 13

-1 7 14

-3 15

-1 7 16

-1 7 21

-1 6 22 

-2 23

-4 5 1 8 0

-1 8 24

-5 20 25

-3 26

-1 8 30

-5 20 35

-6