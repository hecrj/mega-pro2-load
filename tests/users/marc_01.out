Peticion procesada y servidores
0 1
3 
Peticiones pendientes
0
Pelicula mas solicitada
0 0
Peticion procesada y servidores
1 5
5 6 
Pelicula mas solicitada
3 1
Peticion procesada y servidores
2 1
2 
Servidores ocupados
0
Peticion procesada y servidores
3 8
3 8
Peticion procesada y servidores
4 50
1 
Servidores ocupados
1 4 49
3 3 6
8 3 6
Peticion procesada y servidores
5 0
Peticion procesada y servidores
6 8
3 8 
Peticion procesada y servidores
7 0
Peticiones pendientes
6 7 21 29
4 7 14 64
Peticion procesada y servidores
8 15
5 
Pelicula mas solicitada
7 1
Servidores ocupados
1 4 38
3 6 3
5 8 13
8 6 3
Peticion procesada y servidores
9 18
2 
Pelicula mas solicitada
8 2